### ISSUE TITLE
Text in the lattice has outlines intersecting the text.

### ISSUE THREAD

**Issue by matj1** [xenharmonic-devs/scale-workshop#749]
The text in the lattice view has a thin outline, and the font used on Android intersects self, which causes that the outline goes through the characters, which can be seen in the image at the bottom. 

The outline is the SVG stroke of the glyph, so, unless it is a problem the be solved just in Android, I suggest this: Remove the stroke, and either leave it so, or generate a separate slightly thicker path behind the glyph and fill it with the background color.

Reproduction: Outline intersecting the glyph happens on Android (13 and 14), not on Arch Linux. Cromite (Android Chromium) doesn't show the outlines at all, but Arch Chromium does. Or, to bypass all that, force a font which doesn't interest self.

![Screenshot_20240615-080756_Firefox](https://github.com/xenharmonic-devs/scale-workshop/assets/22873928/844f867f-cc69-4149-a56e-4dcfd584fb72)

**Comment by frostburn** [comment]
This is a browser-specific bug, so you might want to take it up with your vendor too.

We can add an option to circumvent it, though.

**Comment by matj1** [comment]
> We can add an option to circumvent it, though.

Yes, all my suggestions above are to circumvent it. I prefer forcing a font which doesn't intersect self because that seems like the least work with almost no aesthetic compromise.

**Comment by frostburn** [comment]
Is this really indentended SVG behavior with those kinds of fonts? Can you provide a font family that does this in all browsers?

**Comment by matj1** [comment]
No, this seems like a bug of Android text rendering. I mean that, if you add an option to force a specific nonselfintersecting font, it will avoid the issue. (circumvention as you mentioned)
